docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,154 @@
+5. All channels managed by the targeted ordering service node must be migrated during the same maintenance window. It is not supported to migrate only some channels before resuming operations.
+6. At the end of the migration process, every channel will have the same consenter set of BFT nodes.
+7. Migration is done in place, utilizing the existing ledgers for the deployed ordering nodes. Addition or removal of orderers should be performed after the migration. 
+
+
+## Migration flow
+Migration is carried out in five phases.
+
+1. The system is placed into a maintenance mode where application transactions are rejected and only ordering service admins can make changes to the channel configuration.
+2. The system is stopped, and a backup is taken in case an error occurs during migration.
+   Comment: backup should be taken at this point since in maintenance mode data tx's are blocked, that's how we ensure that we take a backup of a state which is coherent.
+3. The system is started, and each channel has its consensus type and metadata modified.
+4. The system is restarted and is now operating on BFT consensus; each channel is checked to confirm that it has successfully achieved a quorum.
+5. The system is moved out of maintenance mode back to the normal state and normal function resumes.
+
+
+## Preparing to migrate
+
+The migration process requires changes in the channel configuration at /Channel/Orderer/ConsensusType and /Channel/Orderer/Orderers.
+In a JSON representation of the channel configuration, this would be .channel_group.groups.Orderer.values.ConsensusType and .channel_group.groups.Orderer.values.Orderers respectively.
+
+The `ConsensusType` is represented by three values: `Type`, `Metadata`, and `State`, where:
+- `Type` is either Raft (etcdraft) or BFT. This value can only be changed while in maintenance mode.
+- `Metadata` must carry valid Raft or BFT metadata.
+- `State` is the state of the system and can be one of the following: `STATE_NORMAL`, when the channel is processing transactions, or `STATE_MAINTENANCE` during the migration process.
+
+The `Orderers` is represented by one value, which is `ConsenterMapping`.
+Each consenter is represented by the following properties:
+* id
+* host
+* port
+* msp_id
+* identity
+* client_tls_cert
+* server_tls_cert
+
+\
+Before proceeding with the migration, the following preparatory steps are required:
+
+1.  **Config the number of node to be 3f + 1, so the cluster will be functional.**
+2. Prepare the BFT `Metadata` channel configuration.
+Refer to the [BFT configuration guide](https://hyperledger-fabric.readthedocs.io/en/latest/bft_configuration.html?highlight=BFT#configuring-and-operating-a-bft-ordering-service) for more information on the channel configuration fields.
```

Comment:
```suggestion
Refer to the [BFT configuration guide](./bft_configuration.html) for more information on the channel configuration fields.
```

We use relative links so that the link works on any branch of the documentation.